kiwi/system/setup.py

```diff
@@ -302,8 +299,10 @@ def setup_users(self):
                 user_id = user.get_id()
                 user_realname = user.get_realname()
                 user_shell = user.get_shell()
-                user_secgroups = user.get_secgroups()
-                user_group = user.get_group()
+                if user.get_groups():
+                    user_groups = user.get_groups().split(',')
+                else:
+                    user_groups = None
```

Comment:
This can be oneliner:
```python
user_groups = user.get_groups().split(",") if user.get_groups() else None
```
But as I already said, this code even should not exist. You are splitting groups all around the place. Instead, the `user.get_groups()` should either return `None` or a list of groups, preferably objects (see above the case what for).